ecl/hql/reservedwords.cpp

```diff
@@ -176,7 +181,7 @@ static const char * eclReserved7[] = { //Comparison ops
     "<",
     "<=",
     ">",
-    //">=",     //x.<y>= n  really wants to be processed as x.<y> = n, not x.<y >=
```

Comment:
It wasn't so much the comment I was concerned at losing, but the fact that the comment was explaining why this token was commented out (which it no longer is).